frontend/src/locales/de.json

```diff
@@ -212,7 +212,7 @@
     "scheduleDetails": "Zeitplan Details",
     "scheduleSettings": {
       "availabilitySubHeading": "Lege deine Verfügbarkeit fest",
-      "meetingDetailsSubHeading": "Videos oder Notizen den Events hinzufügen",
+      "meetingDetailsSubHeading": "Videos und/oder Notizen den Events hinzufügen",
```

Comment:
"und/oder" halte ich für redundant. Da hier Möglichkeiten aufgezählt werden, ist das "und" bereits im "oder" enthalten. Es handelt sich ja nicht um ein "entweder oder". Aber das ist nur meine bescheidene Meinung 😇

Reply:
Das kann man so sehen. Mir war nur beim englischsprachigen Text aufgefallen, dass dort „and/or“ steht. Daher dachte ich, dass das in der Übersetzung verloren gegangen ist. Wenn du es immer noch nur mit „oder“ besser findest, ändere ich das gerne noch. Dann wäre vielleicht die Frage, ob man die englischsprachige Version anpassen sollte oder ob der Unterschied egal ist.

Reply:
Ah guter Punkt. Nein, dann lass uns näher am Ursprungstext bleiben. Lass es gern drin. Falls entschieden wird, dass es im Englischen gestrichen oder geändert wird, dann reagieren wir an dieser Stelle. Danke dir.